website/client/js/services/userServices.js

```diff
@@ -633,7 +633,7 @@ angular.module('habitrpg')
           var isStaticOrSocial = $window.location.pathname.match(/^\/(static|social)/);
           if (!isStaticOrSocial){
             localStorage.clear();
-            window.location.href = '/logout';
+            $window.location.href = '/logout';
```

Comment:
yeah I'm trying to find where it happens.. they could provide some more info with that error